README.md

```diff
@@ -28,6 +28,10 @@ Find out more in [`/docs/ardana/`](docs/ardana/)
 
 Find out more in [`/docs/devstack/`](docs/devstack/)
 
+## Manual deployements
```

Comment:
This section is out of place here. You should instead update the existing `Unified Cloud deployments` section that still references the wrong path.